gluu-server/opt/gluu/jetty/oxauth/custom/scripts/person_authentication/SignInCanada.py

```diff
@@ -686,6 +732,8 @@ def getNextStep(self, configurationAttributes, requestParameters, step):
         session = identity.getSessionId()
         rpConfig = self.rputils.getRPConfig(session)
         provider = identity.getWorkingParameter("provider")
+        if provider == "gckeyregister": # Hack
```

Comment:
was this hack needed for this PR or could it be pushed into a new PR and explain the reason?

Reply:
oh I see now, thanks for sharing